src-tauri/src/utils.rs

```diff
@@ -77,3 +80,16 @@ pub(crate) fn custom_log_dir(id: &str) -> PathBuf {
 
     path.expect("log path generation should succeed")
 }
+
+// TODO: current implementation works only on unix like systems
+#[tauri::command]
+pub(crate) fn create_file(path: &str, content: String) {
+    let mut file = File::create(path).expect("can't create file");
+    file.write_all(content.as_bytes())
+        .expect("couldn't write file");
+    let metadata = file.metadata().expect("can't get metadata");
+    let mut permissions = metadata.permissions();
+
+    permissions.set_mode(0o644); // Read/write for owner and read for others.
```

Comment:
why `644`? Do we want it to be readable by the group and public?

Reply:
I think it's better we disable read for others as well, no need for a 3rd party to reach the reward address (although it is not critically harmful, there is simply no need)